src/Sheet/Sheet.scss

```diff
@@ -20,34 +31,38 @@
     color: $light-300;
   }
   &.bottom {
-    bottom: 0px;
-    box-shadow: 0px -4px 10px 0px rgba(0,0,0,0.15);
```

Comment:
I believe this line gets overridden by the next one, unless I'm crazy... removed it.